shell/app-shell/app-shell.js

```diff
@@ -401,6 +409,7 @@ class AppShell extends Xen.Base {
       const url = state.arc._loader._resolve(state.avatar.rawData.url);
       state.avatarStyle = `background: url('${url}') center no-repeat; background-size: cover;`;
     }
+    state.shareStateTitle = [`Private Arc`, `Use For My Suggestions`, `Use For Friends' Suggestions`][state.shareState];
```

Comment:
I'm not sure the review is the right spot for it but I wonder if there's a better term than suggestions here. I think it's accurate but I'm not sure it's clear to a new user.

Reply:
Good question, but yes, I think this question should flow down from design mocks, not up from the code.

Reply:
Sounds good, let's have the discussion there.